quickstarts/microsoft.web/function-app-windows-consumption/main.bicep

```diff
@@ -0,0 +1,125 @@
+@description('The name of the Azure Function app.')
+param functionAppName string = 'func-${uniqueString(resourceGroup().id)}'
+
+@description('Storage Account type')
+@allowed([
+  'Standard_LRS'
+  'Standard_GRS'
+  'Standard_RAGRS'
+])
+param storageAccountType string = 'Standard_LRS'
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+@description('The language worker runtime to load in the function app.')
+@allowed([
+  'dotnet'
+  'node'
+  'python'
+  'java'
+])
+param functionWorkerRuntime string = 'node'
+
+@description('The base URI where artifacts required by this template are located including a trailing \'/\'')
+param _artifactsLocation string = deployment().properties.templateLink.uri
+
+@secure()
+@description('The sasToken required to access _artifactsLocation.  When the template is deployed using the accompanying scripts, a sasToken will be automatically generated. Use the defaultValue if the staging location is not secured.')
+param _artifactsLocationSasToken string = ''
+
+var packageUri = uri(_artifactsLocation, 'packages/nodejs.zip${_artifactsLocationSasToken}')
```

Comment:
@patelchandni we're having trouble deploying with the package .zip file in this repo. Is this possible, or does it have to be some public URL location? I'm not familiar with how to do this in Bicep (or ARM frankly).